src/richie/apps/courses/templates/courses/cms/blogpost_detail.html

```diff
@@ -1,16 +1,15 @@
 {% extends "richie/fullwidth.html" %}
 {% load static category_tags cms_tags extra_tags i18n thumbnail %}
 
-{% block meta_opengraph_type %}
-    <meta property="og:type" content="article">
-{% endblock meta_opengraph_type %}
-
-
-{% block meta_opengraph_contextuals %}
-    <meta property="og:title" content="{{ current_page.get_title|truncatechars:65 }}">
-    <meta property="og:url" content="{{ SITE.web_url }}{{ current_page.get_absolute_url }}">
-    <meta property="article:published_time" content="{{ current_page.publication_date|date:"c" }}">
-{% endblock meta_opengraph_contextuals %}
+{% block meta_rdfa_context %}
+    <meta property="og:type" content="article" />
+    <meta property="og:article:published_time" content="{{ current_page.publication_date|date:"c" }}">
+
```

Comment:
Yes good idea, and also define categories in the schema.org markup. I didn't do it here because it requires some work to retrieve them as a list of keywords. I will update the issue to mark it as TODO.